tests/acceptance/infinity-route-with-meta-test.js

```diff
@@ -48,7 +48,7 @@ module('Acceptance: Infinity Route', {
 });
 
 test('it works when meta is present in payload', assert => {
-  visit('/');
+  visit('/test');
```

Comment:
I re-pathed the route used for acceptance tests, since there's more value in using the root URL `/` for humans